package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-shoppe-demo",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
nit: I feel like some of these dependencies could be devDependencies - like testing features or types.

Reply:
It seems that the CRA team has an opinion on this:

- https://github.com/facebook/create-react-app/pull/2657
- https://github.com/facebook/create-react-app/issues/2696

I'm thinking I'll just stick with the opinion of the CRA team, since they feel a lot stronger about tis than I do :)